### PR TITLE
Tweak github actions to publish for mac arm, change artifact naming

### PR DESCRIPTION
**Pull request by zhrebicek**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -32,12 +32,12 @@ jobs:
...
+          cp target/release/protofetch protofetch_linux_amd64
+          gh release upload $TAG_NAME protofetch_linux_amd64
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
 
-  release-mac:
+  release-mac-amd64:
     runs-on: macos-latest
     steps:
       - name: Checkout sources
```
Comment (saraiva132) [inline .github/workflows/release.yml:43]: I am not familiar with this but. No action required ?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -57,7 +57,33 @@ jobs:
...
         env:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+  release-mac-arm64:
+    runs-on: macos-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install stable toolchain
```
Comment (saraiva132) [inline .github/workflows/release.yml:71]: No reference to arm in this step. Can you elaborate a bit here? So I also understand :D

Comment (zhrebicek) [reply .github/workflows/release.yml:71]: hmm :D I have probably forgot something only changing target :D

File: .github/workflows/release.yml (hunk 1 above)
Comment (zhrebicek) [reply .github/workflows/release.yml:43]: The one without suffix was for amd64

File: .github/workflows/release.yml (hunk 2 above)
Comment (zhrebicek) [reply .github/workflows/release.yml:71]: but then again `protodep` used ubuntu runner on which it builds all targets

**Review by saraiva132 — APPROVED** [review]

**Pull request merged**